samples/snippets/fsharp/VS_Snippets_CLR_System/system.Action/fs/Action.fs

```diff
@@ -0,0 +1,21 @@
+module Action
+
+// <Snippet2>
+open System
+open System.Windows.Forms
+
+type Name(name) =
+    let instanceName = name
```

Comment:
In F# we don't need the explicit local field set; in `DisplayToConsole` and `DisplayToWindow` we are able to use the `name` directly. I'm guessing that this form what chosen to maintain symmetry with the C# samples? I'm not sure that's necessary to be honest.